EIPS/eip-action_standard.md

```diff
@@ -0,0 +1,301 @@
+
+### Benefits
+1. Make interactive token contracts **discoverable and usable** by metaverse/game applications
+2. Allow for generalized action bridges to transmit actions between chains (enabling actions on L1 assets to be saved to L2s, and L1 assets to interact with L2 assets).
+3. Provide a simple solution with viable validity guarantees for developers to make dynamic NFTs and other tokens
+4. Promote decentralized, collaborative game building
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+```solidity
+pragma solidity ^0.8.0;
+
+/// @title ERC-xxxx Token Interaction Standard
+/// @dev See https://eips.ethereum.org/EIPS/eip-xxxx
+interface IERCxxxxSender {
+    /// @notice Send an action to the target address
+    /// @dev The action's `fromContract` is automatically set to `address(this)`,
+    /// and the `from` parameter is set to `msg.sender`.
+    /// @param action The action to send
+    function sendAction(Action memory action) external payable;
+
+    /// @notice Check if an action is valid based on its hash and nonce
+    /// @dev When an action passes through all three possible contracts
+    /// (`fromContract`, `to`, and `state`) the `state` contract validates the
+    /// action with the initating `fromContract` using a nonced action hash.
+    /// This hash is calculated and saved to storage on the `fromContract` before
+    /// action handling is initiated. The `state` contract calculates the hash
+    /// and verifies it and nonce with the `fromContract`.
+    /// @param _hash The hash to validate
+    /// @param _nonce The nonce to validate
+    function isValid(uint256 _hash, uint256 _nonce) external returns (bool);
+
+    /// @notice Retrieve list of actions that can be sent.
+    /// @dev Intended for use by off-chain applications to query compatible contracts.
+    function sendableActions() external view returns (bytes4[]);
+
+    /// @notice Change or reaffirm the approved address for an action
+    /// @dev The zero address indicates there is no approved address.
+    ///  Throws unless `msg.sender` is the `_account`, or an authorized
+    ///  operator of the `_account`.
+    /// @param _account The account of the account-action pair to approve
+    /// @param _action The action of the account-action pair to approve
+    /// @param _approved The new approved account-action controller
+    function approveForAction(
+        address _account,
+        bytes4 _action,
+        address _approved
+    ) external returns (bool);
+
+    /// @notice Enable or disable approval for a third party ("operator") to conduct
+    ///  all actions on behalf of `msg.sender`
+    /// @dev Emits the ApprovalForAll event. The contract MUST allow
+    ///  multiple operators per owner.
```

Comment:
Is allowing at most two operators compliant with this specification? If not, I'd recommend:

```suggestion
    /// @dev Emits the ApprovalForAll event. The contract MUST allow
    ///  an unbounded number of operators per owner.
```